modules/cluster-logging-rn-5.7.1.adoc

```diff
@@ -0,0 +1,27 @@
+
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-7-1_{context}"]
+= Logging 5.7.1
+This release includes: link:https://access.redhat.com/errata/RHBA-2023:3197[OpenShift Logging Bug Fix Release 5.7.1].
+
+[id="openshift-logging-5-7-1-bug-fixes"]
+== Bug fixes
+* Before this update, the presence of numerous noisy messages within Cluster Logging Operator pod logs caused reduced log readability and increased difficulty in identifying important system events. With this update, the issue is resolved by significantly reducing the noisy messages within Cluster Logging Operator pod logs, resulting in improved log readability and enhanced visibility of critical system events. (link:https://issues.redhat.com/browse/LOG-3482[LOG-3482])
+
+* Before this update, the API server would default the new collector type field to "vector" even when the custom resource had the prior field selected. This update removes the default for the new collector type field to restore the previous behavior. (link:https://issues.redhat.com/browse/LOG-4086[LOG-4086])
+
+* Before this update, within the {Product-Title} web console a time range could not be selected by click dragging over the logs histogram. With this update, click dragging can successfully select a time range. (link:https://issues.redhat.com/browse/LOG-4501[LOG-4501])
```

Comment:
```suggestion
* Before this update, a time range could not be selected in the {Product-Title} web console by clicking and dragging over the logs histogram. With this update, clicking and dragging can be used to successfully select a time range. (link:https://issues.redhat.com/browse/LOG-4501[LOG-4501])
```

suggested editing for clarity

Reply:
not positive wrt. "click dragging" as a term, but per https://www.ibm.com/docs/en/ibm-style?topic=word-usage I'd probably spell it out like this to be safe - might be worth asking core OCP team for advice or looking for other instances in the docs. By doing a quick search, it looking like "clicking and dragging" is used in other places, e.g. https://docs.openshift.com/container-platform/4.13/monitoring/querying-metrics.html#exploring-the-visualized-metrics_querying-metrics